nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2944,95 +3048,95 @@ at this time.
+|`nifi.provenance.repository.journal.count`|The number of journal files that should be used to serialize Provenance Event data. Increasing this value will allow more tasks to simultaneously update the repository but will result in more expensive merging of the journal files later. This value should ideally be equal to the number of threads that are expected to update the repository simultaneously, but 16 tends to work well in must environments. The default value is `16`.
+|`nifi.provenance.repository.indexed.fields`|This is a comma-separated list of the fields that should be indexed and made searchable. Fields that are not indexed will not be searchable. Valid fields are: `EventType`, `FlowFileUUID`, `Filename`, `TransitURI`, `ProcessorID`, `AlternateIdentifierURI`, `Relationship`, `Details`. The default value is: `EventType, FlowFileUUID, Filename, ProcessorID`.
+|`nifi.provenance.repository.indexed.attributes`|This is a comma-separated list of FlowFile Attributes that should be indexed and made searchable. It is blank by default.  But some good examples to consider are `filename`, `uuid`, and `mime.type` as well as any custom attritubes you might use which are valuable for your use case.
+|`nifi.provenance.repository.index.shard.size`|Large values for the shard size will result in more Java heap usage when searching the Provenance Repository but should provide better performance. The default value is `500 MB`.
+|`nifi.provenance.repository.max.attribute.length`|Indicates the maximum length that a FlowFile attribute can be when retrieving a Provenance Event from the repository. If the length of any attribute exceeds this value, it will be truncated when the event is retrieved. The default value is `65536`.
 |====
 
 === Volatile Provenance Repository Properties
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.buffer.size|The Provenance Repository buffer size. The default value is `100000`.
+|`nifi.provenance.repository.buffer.size`|The Provenance Repository buffer size. The default value is `100000`.
 |====
 
 === Write Ahead Provenance Repository Properties
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.directory.default*|The location of the Provenance Repository. The default value is `./provenance_repository`. +
+|`nifi.provenance.repository.directory.default`*|The location of the Provenance Repository. The default value is `./provenance_repository`. +
  +
 	*NOTE*: Multiple provenance repositories can be specified by using the *_nifi.provenance.repository.directory._* prefix with unique suffixes and separate paths as values. +
  +
 	For example, to provide two additional locations to act as part of the provenance repository, a user could also specify additional properties with keys of: +
  +
-	nifi.provenance.repository.directory.provenance1=/repos/provenance1 +
-	nifi.provenance.repository.directory.provenance2=/repos/provenance2 +
+	`nifi.provenance.repository.directory.provenance1`=/repos/provenance1 +
```

Comment:
These two lines should be `code formatted` for the entire line, as the value is also code.

Reply:
Good catch.  Missed that.